.github/workflows/pipeline.yaml

```diff
@@ -0,0 +1,76 @@
+name: Azure Pipeline Merge Queue Check
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+    - master
```

Comment:
should this added to `release` branches too ?

Reply:
We don't need to add this to release/* branches as they do not use merge queue to merge.